gateway/jest.config.js

```diff
@@ -0,0 +1,64 @@
+module.exports = {
+	// Test environment
+	testEnvironment: "jsdom",
+
+	// Test file patterns
+	testMatch: [
+		"**/sds_gateway/static/js/**/*.test.js",
+		"**/sds_gateway/static/js/**/*.spec.js",
+		"**/sds_gateway/static/js/tests/**/*.js",
+	],
+
+	// Files to collect coverage from
+	collectCoverageFrom: [
+		"sds_gateway/static/js/**/*.js",
+		"!sds_gateway/static/js/tests/**",
+		"!sds_gateway/static/js/**/*.test.js",
+		"!sds_gateway/static/js/**/*.spec.js",
+		"!sds_gateway/static/js/**/node_modules/**",
+	],
+
+	// Coverage thresholds
+	coverageThreshold: {
+		global: {
+			branches: 70,
+			functions: 70,
+			lines: 70,
+			statements: 70,
+		},
+	},
+
+	// Coverage reporters
+	coverageReporters: ["text", "html", "lcov"],
+
+	// Coverage directory
+	coverageDirectory: "coverage",
```

Comment:
This root dir is already a bit cluttered. Maybe we can tuck these in `sds_gateway/static/js/tests-config/.coverage` + git-ignore `.coverage` if not already doing that.